PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Remove Extra Bitlocker Recovery Passwords.md

```diff
@@ -0,0 +1,73 @@
+#### Name
+
+Windows - Remove Extra Bitlocker Recovery Passwords | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```powershell
+# Set this variable to the password recovery key to PRESERVE.  All keys other than this key will be deleted.
+$RecoveryKey = 'RECOVERY KEY TO PRESERVE'
+
+function Get-OsVolumeLetter {
+  return Get-WmiObject -Class Win32_OperatingSystem -Property SystemDrive | Select-Object -ExpandProperty SystemDrive
+}
+
+function Remove-NonMatchingBitLockerRecoveryPasswords {
+  [CmdletBinding()]
+  param(
+    [Parameter(Mandatory=$true)]
+    [ValidateNotNull()]
+    [string] $RecoveryKey
+  )
+
+  try {
+	$osVolumeLetter = Get-OsVolumeLetter
```

Comment:
tabbing seems to be off in when displayed in the rich diff.  Not a crazy problem, but could make the code a bit hard to read through

Reply:
Good catch.